File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
       int chars[] = new int[26];
       int strlen = s.length(); 
       for(int i=0;i<strlen;i++){
        chars[s.charAt(i)-'a']++;
       }
       for(int i=0;i<strlen;i++){
        if(chars[s.charAt(i)-'a']==1){
            return i;
        }
       }
       return -1;
    }
}

File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        int chars[26];
        for(char c:s){
            chars[c-'a']++;
        }
        for(int i=0;i<s.size();i++){
            if(chars[s[i]-'a']==1){
                return i;
            }
        }
        return -1;
    }
};